vpr/src/base/vpr_api.cpp

```diff
@@ -502,6 +503,8 @@ bool vpr_pack_flow(t_vpr_setup& vpr_setup, const t_arch& arch) {
 
     if (packer_opts.doPacking == STAGE_SKIP) {
```

Comment:
what does STAGE_SKIP mean?  If it means we don't do a packing or load a packing (just stop the flow) then we probably don't need to run check_cluster_constraints_init() as we didn't actually load or create a packing, so we can't proceed anyway.